content-src/components/TimelinePage/TimelinePage.js

```diff
@@ -17,13 +17,11 @@ const TimelinePage = React.createClass({
     return (<main className="timeline">
       <nav className="sidebar">
         <ul>
-          {navItems.map(item => {
-            return (<li key={item.title}>
-              <a className={item.active ? "active" : ""}>
-                <span className={`fa ${item.icon}`} /> {item.title}
-              </a>
-            </li>);
-          })}
+          {navItems.map((item) => (<li key={item.title}>
+            <a className={item.active ? "active" : ""}>
+              <span className={`fa ${ item.icon }`} /> {item.title}
+            </a>
+          </li>))}
```

Comment:
I forget what rule it was, but ESLint was complaining that we were using a `return` statement as the only statement in an arrow function and it preferred this marginally less verbose syntax. :shrug: